### PR TITLE
Temporary replacement for the inspect menu

### PR DESCRIPTION
**Pull request by mzegar**
Depends on https://github.com/TTTReborn/tttreborn/pull/213

image

### CODE REVIEW

File: code/ui/generalhud/inspectmenu/InspectMenu.scss [hunk 1]
```diff
@@ -1,119 +1,78 @@
...
+    background-color: #212121;
     color: #fafafa;
-    font-family: Poppins;
+    border-radius: 5px;
     font-weight: 500;
-    border-radius: 4px;
-    margin: auto;
-    top: 66%;
+    top: 60%;
+    left: 39.5%;
```
Comment (mzegar) [inline code/ui/generalhud/inspectmenu/InspectMenu.scss:7]: Remind me to put up an issue to center this correctly, we need to wait for the `translate` `overflow` fix.

**Comment by Alf21** [comment]
Problem with vertical scrolling: You've less space displaying stuff, if you'd keep it as it is in TTT1/2, the user would be able to see more information without the need to scroll

**Comment by mzegar** [comment]
> Problem with vertical scrolling: You've less space displaying stuff, if you'd keep it as it is in TTT1/2, the user would be able to see more information without the need to scroll

Agreed, we are stuck with vertical scrolling until those transform overflow bugs are fixed. I definitely want to convert this to horizontal once it's possible in engine.

**Comment by mzegar** [comment]
Closed, merging with UI rework branch.

**Pull request closed without merging**